<?xml version="1.0" ?>
<launch xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xsi:noNamespaceSchemaLocation="https://gist.githubusercontent.com/nalt/dfa2abc9d2e3ae4feb82ca5608090387/raw/roslaunch.xsd">
    <arg name="log_level" default="INFO" />
    <arg name="credentials_path" default="~/.config/niryo"/>

    <node name="niryo_robot_credentials" pkg="niryo_robot_credentials" type="credentials_node.py" output="screen">
        <param name="serial_number_file_path" value="$(arg credentials_path)/serial_number_do_not_touch"/>
        <param name="api_key_file_path" value="$(arg credentials_path)/api_key_do_not_touch"/>
        <param name="log_level" type="string" value="$(arg log_level)" />
    </node>
</launch>
